sovrin_client/test/agent/acme.py

```diff
@@ -154,10 +154,7 @@ def getAvailableClaimList(self, requesterId):
         return schemaJobCert
 
     async def bootstrap(self):
-        ranViaScript = False
-        if __name__ == "__main__":
-            ranViaScript = True
-        isSchemaFound(await self.addSchemasToWallet(), ranViaScript)
+        await runBootstap(isMain(), self.addSchemasToWallet)
```

Comment:
Feels strange... we shouldn't be referencing main from within the class. Let's leave this outside.